tests/builders/test_rnn.py

```diff
@@ -90,36 +69,6 @@ def test_build_rnn_returns_correct_rnn_with_valid_params(
     rnn_match_cfg(rnn, rnn_cfg, input_features, batch_first)
 
 
-@given(rnn_cfg_tensor=rnn_cfg_tensors())
-def test_build_rnn_rnn_forward_output_correct_size(
-    rnn_cfg_tensor: Tuple[torch.nn.Module, rnn_pb2.RNN, torch.Tensor, bool]
-) -> None:
-    """Ensures returned RNN forward produces output with correct size."""
-    rnn, rnn_cfg, tensor, batch_first = rnn_cfg_tensor
-    if batch_first:
-        batch, seq_len, input_features = tensor.size()
-    else:
-        seq_len, batch, input_features = tensor.size()
-
-    in_seq_lens = torch.randint(low=1, high=1 + seq_len, size=(batch,))
-
-    out, out_seq_lens = rnn((tensor, in_seq_lens))
-
-    if batch_first:
-        out_batch, out_seq_len, out_features = out.size()
-    else:
-        out_seq_len, out_batch, out_features = out.size()
-
-    assert out_seq_len == seq_len
-    assert out_batch == batch
-    expected_out_features = rnn_cfg.hidden_size
-    if rnn_cfg.bidirectional:
-        expected_out_features *= 2
-    assert out_features == expected_out_features
-
-    assert torch.all(in_seq_lens == out_seq_lens.to(in_seq_lens.device))
-
-
```

Comment:
I realised that the functionality in this test is exactly duplicated in  `tests/model/test_rnn.py::test_rnn_forward_pass_correct_shapes_returned`